File: 0268-missing-number/0268-missing-number.py
class Solution(object):
    def missingNumber(self, nums):
       length=len(nums)
       sum1=0
       sum2=0
       ans=0
       for i in range(0,length+1):
        sum1+=i
       sum2=sum(nums)
       ans=sum1-sum2
       return ans
         


        

        

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int len=nums.length;
        int xor=0;
        int xor1=0;
        for(int i=0;i<=len;i++)
        {
            xor=xor^i;
        }

        for(int i=0;i<len;i++)
        {
            xor1=xor1^nums[i];
        }

        return (xor^xor1);
        
    }
}